### PR TITLE
Reorganizes the docs

### PR DESCRIPTION
**Pull request by ignasi35**
This PR is built on top of #40

### CODE REVIEW

**Comment by ignasi35** [comment]
review https://github.com/playframework/play-scala-grpc-example/pull/40#discussion_r253398138

File: docs/src/main/paradox/code-details.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+```scala
+import akka.stream.Materializer
+import example.myapp.helloworld.grpc.{HelloReply, HelloRequest}
+import javax.inject.Inject
+
+import scala.concurrent.Future
+
+
+class HomeController @Inject() (mat: Materializer, greeterServiceClient: GreeterServiceClient) extends InjectedController {
+  implicit val ec = mat.executionContext
```
Comment (marcospereira) [inline docs/src/main/paradox/code-details.md]: Why not inject the `ExecutionContext` instead?

File: docs/src/main/paradox/locally.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. you may use `sbt runProd` to run Play locally in a forked JVM in PROD mode, or
+1. you may use `./ssl-play run` to run Play in DEV mode within `sbt`.
+
+`ssl-play` is a wrapper script around `sbt` that sets up the ALPN agent (required for HTTP/2) on the JVM running `sbt`.  
+
+In both execution modes above, `sbt` will also generate the server and client sources based on the `app/proto/*.proto` 
+files, which happens thanks to the Akka gRPC plugin being enabled. 
+
+Finally, for your convenience, a self-signed certificate for 
+`CN='localhost' is provided in this example (see `conf/selfsigned.keystore`). Setting
```
Comment (marcospereira) [inline docs/src/main/paradox/locally.md]:
```suggestion
`CN='localhost'` is provided in this example (see `conf/selfsigned.keystore`). Setting 
```

File: docs/src/main/paradox/locally.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+In both execution modes above, `sbt` will also generate the server and client sources based on the `app/proto/*.proto` 
+files, which happens thanks to the Akka gRPC plugin being enabled. 
+
+Finally, for your convenience, a self-signed certificate for 
+`CN='localhost' is provided in this example (see `conf/selfsigned.keystore`). Setting 
+up a keystore works different in DEV mode and PROD mode. Locate the `play.server.https.keyStore.path` setting in 
+`application.conf` and `build.sbt` for an example on how to set the keystore on each environment.
+
+## Limitations
```
Comment (marcospereira) [inline docs/src/main/paradox/locally.md]: This section is duplicating the text above.

**Review by marcospereira — COMMENTED** [review]
Some minor details.

File: docs/src/main/paradox/code-details.md (hunk 1 above)
Comment (ignasi35) [reply docs/src/main/paradox/code-details.md]: Ah, forgot to fix the docs. This was copy/pasted from the code and I had it fixed there.
Actually, I'll make this snippet injected via paradox code embedding.

File: docs/src/main/paradox/locally.md (hunk 3 above)
Comment (ignasi35) [reply docs/src/main/paradox/locally.md]: Good catch, thanks! :-D

**Review by marcospereira — APPROVED** [review]
LGTM.

**Comment by ignasi35** [comment]
Updating the `akka-grpc` version from `0.4.2` to `0.5.0` causes a compilation error.

**Comment by ignasi35** [comment]
Related: https://github.com/akka/akka-grpc/issues/523

I'll rollback 4f036f6c1b9e6c54a7dcd4c88fbe1bd433a066b3

**Pull request merged**